build/azure-pipelines/build-product.yml

```diff
@@ -74,7 +74,8 @@ stages:
       pool:
         vmImage: 'windows-latest'
       variables:
-        versionTag: $[ stageDependencies.Compile.Sqlcmd.outputs['getVersion.VERSION_TAG'] ]
+        - name: versionTag

```

Comment:
what's different about this syntax that makes it work?

Reply:
This was just to see if it works. Have added isOutput=true to export the variable.